server/src/main/java/io/crate/analyze/TableInfoToAST.java

```diff
@@ -250,7 +250,9 @@ private PrimaryKeyConstraint<Expression> extractPrimaryKeyConstraint() {
             if (tableInfo.primaryKey().size() == 1 && tableInfo.primaryKey().get(0).isSystemColumn()) {
                 return null;
             }
-            return new PrimaryKeyConstraint<>(tableInfo.pkConstraintName(), expressionsFromColumns(tableInfo.primaryKey()));
+            return new PrimaryKeyConstraint<>(
+                tableInfo.pkConstraintNameOrDefault(),
```

Comment:
Isn't it important here to keep setting it to `null` to avoid `SHOW CREATE TABLE` from showing the implicit constraint name as if it were provided as explicit name?